text/0000-lts-releases.md

```diff
@@ -0,0 +1,109 @@
+# Meta
+[meta]: #meta
+- Name: Lifecycle releases with long term support (LTS)
+- Start Date: 2023-02-24
+- Author(s): joe-kimmel-vmw
+- Status: Draft <!-- Acceptable values: Draft, Approved, On Hold, Superseded -->
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: N/A
+
+# Summary
+[summary]: #summary
+
+Lifecycle to define approximately 4 releases per year as LTS.
```

Comment:
This seems fairly high. I don't think we actually do enough minor releases an year that picking 4 would help. This would roughly result in every lifecycle minor being an LTS. I would be more comfortable if we reduce this cadence.

I also am not sure how much benefit this provides us given lifecycle is supposed to provide strong backwards compatibility guarantees.

Reply:
If the argument were instead made for one lifecycle release every 1 year (or possibly every 2 years) being marked as LTS, this would make more sense to me.

Reply:
I agree with Sam. I think 1-2 LTS releases a year seem like plenty.

Reply:
Thanks - that makes sense